starter-workflows/code-scanning/frogbot-fix.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
```

Comment:
Let's add here the following comment:

Read morte about Frogbot here - https://github.com/jfrog/frogbot#frogbot

The same goes for the other template.

Reply:
done